broker/broker.go

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package broker
+
+import (
+	"context"
+
+	"github.com/nats-io/nats.go"
+)
+
+// SubjectAllChannels allows to subscribe to all subjects of all channels
+const SubjectAllChannels = "channel.>"
+
+// Nats specifies a NATS message API.
+type Nats interface {
+	// Publish publishes message to the msessage broker.
+	Publish(context.Context, string, Message) error
```

Comment:
Can you pls put named params, it's easier to understand what are params. Now I see `string string` and dunno what it means.

Reply:
I think that's what we should do for all the interfaces, starting from here.

Reply:
I'm not against but in most of our services we are not doing this. Let me open an issue to fix it everywhere